pjsip-apps/src/pjsua/pjsua_app_config.c

```diff
@@ -562,6 +567,10 @@ static pj_status_t parse_args(int argc, char *argv[],
         { "vrender-dev",  1, 0, OPT_VRENDER_DEV},
         { "play-avi",   1, 0, OPT_PLAY_AVI},
         { "auto-play-avi", 0, 0, OPT_AUTO_PLAY_AVI},
+        { "rec-avi",   1, 0, OPT_REC_AVI},
```

Comment:
`play-avi` can be specified multiple times to play multiple AVI files, while it does not seem to be the same with `rec-avi`.
I guess it need to be described somewhere.

Reply:
Looking at the other commands (such as for WAV --play-file and --rec-file), it seems that the description should be put in the command that can be specified multiple times, so I follow that pattern in the last commit.